src/XMakeBuildEngine/Definition/ToolsetReader.cs

```diff
@@ -681,24 +680,24 @@ private MSBuildExtensionsPathReferenceKind(string value)
         /// <summary>
         /// Tries to find a reference to MSBuildExtensionsPath* property in the given string
         /// </summary>
-        public static MSBuildExtensionsPathReferenceKind FindIn(string expression)
+        public static string FindIn(string expression)
```

Comment:
This is used only in `ExpandAndLoadImports`, which should be fine with this method returning `MSBuildExtensionsPathReferenceKind`.